lib/rust/ensogl/core/src/display/world.rs

```diff
@@ -243,7 +243,7 @@ impl WorldData {
             let event = val.unchecked_into::<web::KeyboardEvent>();
             if event.alt_key() && event.ctrl_key() {
                 let key = event.code();
-                if key == "Backquote" {
+                if key == "Backquote" || key == "IntlBackslash" {
```

Comment:
Please explain in the comment what this key is mapped to on your system and why its needed.